tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -219,18 +221,57 @@ def _create_methodologies_snapshot(self) -> str:
             themes.append(theme)
         return json.dumps(themes, sort_keys=True, indent=2)
 
-    def validate_snapshots(self):
+    def _build_slack_blocks(self, messages: List[str]) -> List | None:
+        # if length of messages is 0, then no errors were found
+        blocks = []
+        if len(messages) > 0:
+            # PR is raised due to snapshot mismatch
+            blocks.append(
+                {
+                    "type": "header",
+                    "text": {
+                        "type": "plain_text",
+                        "text": "Snapshots do not match . See pull request for more details",
+                    },
+                }
+            )
+
+            blocks.append(
+                {
+                    "type": "actions",
+                    "elements": [
+                        {
+                            "type": "button",
+                            "style": "danger",
+                            "text": {"type": "plain_text", "text": "View pull request"},
+                            "url": "https://github.com/dfe-analytical-services/explore-education-statistics/pulls/dfe-sdt",
+                        }
+                    ],
+                }
+            )
+        return blocks if len(blocks) > 0 else None
```

Comment:
I think this function can be removed, as it is only called once, and `len(messages) > 0` has already been checked when it's called. So instead on L266 you could have this instead:

```
self._send_slack_notification(blocks=[{
  "type": "header",
  ...
},
{
  "type": "actions",
  ...
}]
```